classes/player/player.gd

```diff
@@ -913,7 +917,11 @@ func ground_failsafe_condition() -> bool:
 func player_fall() -> void:
 	var fall_adjust = vel.y # used to adjust downward acceleration to account for framerate difference
 	if state == S.POUND and pound_state == Pound.SPIN:
+		# Don't move during the pound spin.
 		vel = Vector2.ZERO
+		# A little rising during the wind-up makes it look real nice.
+		if pound_spin_frames <= POUND_SPIN_RISE_TIME:
+			vel.y -= POUND_SPIN_RISE_AMOUNT
```

Comment:
Since vel is set to Vector2.ZERO a few lines before, this can probably just be `vel.y =` instead of `vel.y -=`

Reply:
While you're not wrong, that's an incredible nitpick. If it's not hurting readability, I'm inclined to just leave that.

Reply:
You know what, yeah, your suggestion is easier to understand.